azext_iot/central/params.py

```diff
@@ -654,3 +661,261 @@ def load_central_arguments(self, _):
             options_list=["--children-ids"],
             help="Space-separated list of children device ids.",
         )
+
+    with self.argument_context("iot central enrollment-group") as context:
+        context.argument(
+            "group_id",
+            options_list=["--group-id", "--id"],
+            help="Unique identifier for the enrollment group.",
+        )
+        context.argument(
+            "entry",
+            options_list=["--entry"],
+            help="Entry of certificate only support primary and secondary.",
+        )
+
+    with self.argument_context("iot central enrollment-group create") as context:
+        context.argument(
+            "attestation",
+            options_list=["--attestation", "--at"],
+            help="The attestation mechanism for the enrollment group. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example:./path/to/attestation.json]"
+            " [Example of stringified JSON:[{<Attestation Data JSON>}]."
+            " The request body must contain GroupSymmetricKeyAttestation or GroupX509Attestation.",
```

Comment:
This is copy pasted from the swagger no? Also we need to try to align the arguments with dps commands - see `az iot dps enrollment create`, specifically `--attestation-type`